scenes/game/local_game.gd

```diff
@@ -5994,6 +6083,31 @@ func handle_strike_effect(card_id : int, effect, performing_player : Player):
 			if not nowhere_to_pull:
 				change_game_state(Enums.GameState.GameState_PlayerDecision)
 				events += [create_event(Enums.EventType.EventType_ChooseArenaLocationForEffect, performing_player.my_id, 0)]
+		"pull_to_range":
```

Comment:
make sure you have a test that is like, treasure at location 3, wide tinker at 4/5/6, and you're trying to do this effect.

Reply:
have a set of tinker tank tests yup, though I should probably also check when treasure's at location 4